Sources/FoundationInternationalization/Formatting/Date/ICURelativeDateFormatter.swift

```diff
@@ -73,9 +79,10 @@ internal final class ICURelativeDateFormatter {
         }
     }
 
-    internal static func formatterCreateIfNeeded(format: Date.RelativeFormatStyle) -> ICURelativeDateFormatter {
-        let formatter = Self.cache.formatter(for: format) {
-            ICURelativeDateFormatter(uNumberFormatStyle: format.unitsStyle.icuNumberFormatStyle, uRelDateStyle: format.unitsStyle.icuRelativeDateStyle, locale: format.locale, context: format.capitalizationContext.icuContext)
+    internal static func formatter(for style: Date.RelativeFormatStyle) -> ICURelativeDateFormatter {
+        let signature = Signature(localeIdentifier: style.locale.identifier, numberFormatStyle: style.unitsStyle.icuNumberFormatStyle?.rawValue, relativeDateStyle: style.unitsStyle.icuRelativeDateStyle.rawValue, context: style.capitalizationContext.icuContext.rawValue)
```

Comment:
Should this be the preference-capturing locale identifier to account for, say, custom first weekday? I imagine that information would be useful for calculating relative date in terms of weeks

Reply:
Hm, that may be another bug. rdar://119343432 (Check if ICURelativeDateFormatter should be using preference-capturing locale)